erpnext/manufacturing/doctype/bom/bom_item_preview.html

```diff
@@ -16,26 +16,15 @@ <h4>
 			</div>
 			<hr style="margin: 15px -15px;">
 			<p>
-				{% if data.value %}
+				{% if data.value != "BOM" %}
```

Comment:
This is now showing button for child item without BOM too. It should keep previous check and add this new one IMO. 
```suggestion
				{% if data.value && data.value != "BOM" %}
```